docs/DMS/DMS-AUTH.md

```diff
@@ -37,62 +37,89 @@ update is authorized.
 
 ### 1. Retrieve Api Client Identity from Token
 
-When an API request comes in, the system extracts the identity of the API client by decoding the token included in the request.
+When an API request comes in, the system extracts the identity of the API client
+by decoding the token included in the request.
 
 ### 2. Retrieve associated Claimset
 
-The associated claimset for the Api client will be retrieved. This claimset defines Resource claims, Resource claims actions, Resource claims action authorization strategies.
+The associated claimset for the Api client will be retrieved. This claimset
+defines Resource claims, Resource claims actions, Resource claims action
+authorization strategies.
 
 ### 3. Retrieve and cache Basic Claims details
 
-The system retrieves additional basic claims details, such as the education organization id, key, profiles, and namespace prefixes for the ApiClient. These details are stored in a cached context.
+The system retrieves additional basic claims details, such as the education
+organization id, key, profiles, and namespace prefixes for the ApiClient. These
+details are stored in a cached context.
 
 ### 4. Retrieve Claimset details from Security Database (Cached)
 
-The claimset details, including Resource claims, Resource claims actions, Resource claims action authorization strategies, are retrieved from a security database.
+The claimset details, including Resource claims, Resource claims actions,
+Resource claims action authorization strategies, are retrieved from a security
+database.
 
 ### 5. Check incoming request against Api Details Context
 
-The system checks whether the incoming request complies with the permissions and access rights defined in the API details context. Specifically, the system verifies if the client has access to the requested educational organization or namespace prefix.
+The system checks whether the incoming request complies with the permissions and
+access rights defined in the API details context. Specifically, the system
+verifies if the client has access to the requested educational organization or
+namespace prefix.
 
 ### 6. Check incoming request against Claim Set details
 
-The incoming resource and action will be compared to the resource claims in the claim set. If the resource name is present in the resource claims list, the action will be validated according to the specified authorization strategies. Finally, the resource data will be checked to ensure compliance with the authorization strategy rule sets. If all verification steps pass, the requested action will proceed.
+The incoming resource and action will be compared to the resource claims in the
+claim set. If the resource name is present in the resource claims list, the
+action will be validated according to the specified authorization strategies.
+Finally, the resource data will be checked to ensure compliance with the
+authorization strategy rule sets. If all verification steps pass, the requested
```

Comment:
Can you add a paragraph about these authorization strategy rules, for those who aren't already familiar with how the ODS/API handles this?